hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,7 @@
+RELEASE_TYPE: patch
+
+This patch ensures that, when the ``generate`` :obj:`~hypothesis.settings.phase` is disabled,
```

Comment:
```suggestion
This patch ensures that, when the ``generate`` :obj:`~hypothesis.settings.phases` is disabled,
```

:hand_over_mouth: whoops... this typo was my fault :disappointed_relieved:

Reply:
cdf51943dceb27452900c1e5ded97369a4d4d1d2 Haha should have cross checked with the codebase.